core/src/main/scala/kafka/coordinator/transaction/TransactionMarkerRequestCompletionHandler.scala

```diff
@@ -155,6 +155,14 @@ class TransactionMarkerRequestCompletionHandler(brokerId: Int,
                       txnMarkerChannelManager.removeMarkersForTxnId(transactionalId)
                       abortSending = true
 
+                    case Errors.UNSUPPORTED_FOR_MESSAGE_FORMAT =>
+                      // The producer would have failed to send data to the failed topic so we can safely remove the partition
+                      // from the set waiting for markers
+                      warn(s"Sending $transactionalId's transaction marker from partition $topicPartition has failed with " +
```

Comment:
I'd suggest we log it as `info` instead of `warn`, since for the case when the topic gets deleted TC just log it as `info` while giving up sending markers. I think the case of `topic get deleted` and `topic is on lower version` are of similar alert-importance.